prow/config/config.go

```diff
@@ -174,9 +174,32 @@ type Plank struct {
 	// DefaultDecorationConfig are defaults for shared fields for ProwJobs
 	// that request to have their PodSpecs decorated
 	DefaultDecorationConfig *prowapi.DecorationConfig `json:"default_decoration_config,omitempty"`
+	// Deprecated, use JobURLPrefixConfig instead
 	// JobURLPrefix is the host and path prefix under
 	// which job details will be viewable
+	// TODO @alvaroaleman: Remove in September 2019
 	JobURLPrefix string `json:"job_url_prefix,omitempty"`
+	// JobURLPrefixConfig is the host and path prefix under which job details
+	// will be viewable. Use `*` as key for your global default.
+	JobURLPrefixConfig map[string]JobURLPrefixConfig `json:"job_url_prefix_config,omitempty"`
```

Comment:
Please indicate what this map is keyed by in the comment. As is, this is not enough information for me to provide a valid configuration.

Reply:
Updated